wandb/sdk/internal/internal_api.py

```diff
@@ -887,7 +888,18 @@ def upsert_run(
             }
         }
         """
+
+        replace_string_1 = "$summaryMetrics: JSONString,"
+        replace_string_2 = "summaryMetrics: $summaryMetrics,"
+        mutation_11_0 = mutation_str.replace(
+            replace_string_1, replace_string_1 + "\n            $runQueueItemId: String"
+        ).replace(
+            replace_string_2,
+            replace_string_2 + "\n                runQueueItemId: $runQueueItemId",
         )
+
+        mutation_pre_11_0 = mutation_str
```

Comment:
@kptkin recently found a better way to check gql compat. Basically you can query the graphql schema to see if it has the field you want to send.

Reply:
@kptkin Can you point me to an example of this so I can potentially duplicate the pattern here?

Reply:
@raubitsj I remember you mentioning that it may be best to avoid querying the schema, am I mis remembering that?

Reply:
no, schema reading is best practice these days.   we need to generalize some of the introspection we are doing, but it is ready to have more users.   there are a bit of details about how the introspection is done which probably differ based on use case.

Reply:
great that is awesome, that will really clean things up here !

Reply:
this is fixed